CoinDock-app/src/Shared/Chart/LineChart/LineChart.js

```diff
@@ -0,0 +1,183 @@
+import { React, useState } from "react";
+import moment from "moment";
+import Loading from "Shared/Loading/Loading";
+import "./LineChart.css";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+import { Line } from "react-chartjs-2";
+import {
+  useCoinFilter,
+  useLineChart,
+  useLineFilter,
+} from "App/Api/linechartapi";
+import { isEmpty, uniq } from "lodash";
+import { Card } from "react-bootstrap";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Legend
+);
+const generateRandomColor = () => {
```

Comment:
Can we move this to helper functions in the shared folder since same is being used multiple places